spectrum-network/tests/peer_manager/peer_state_tests.rs

```diff
@@ -1,51 +1,61 @@
 use libp2p::PeerId;
```

Comment:
If you've already had a look in https://github.com/spectrum-finance/spectrum/pull/11, then note that the changes in this file are identical to the changes in the referenced PR (need this for tests to compile).